app/main/index.js

```diff
@@ -45,6 +45,9 @@ app.on('ready', async () => {
     minWidth: 640,
     minHeight: 480,
     show: false,
+    webPreferences: {
+      nodeIntegration: true,
```

Comment:
Node support is off by default in `BrowserWindow`, but we use `require` statements, so we have to enable it, otherwise old school `<script src=..>` way of living